api/v1alpha3/awsmachine_webhook.go

```diff
@@ -183,3 +184,14 @@ func (r *AWSMachine) Default() {
 		r.Spec.CloudInit.SecureSecretsBackend = SecretBackendSecretsManager
 	}
 }
+
+func (r *AWSMachine) validateAdditionalSecurityGroups() field.ErrorList {
+	var allErrs field.ErrorList
+
+	for _, additionalSecurityGroups := range r.Spec.AdditionalSecurityGroups {
+		if len(additionalSecurityGroups.Filters) > 0 {
+			allErrs = append(allErrs, field.Forbidden(field.NewPath("spec.additionalSecurityGroups"), "additional security groups should not have filters"))
```

Comment:
```suggestion
			allErrs = append(allErrs, field.Forbidden(field.NewPath("spec.additionalSecurityGroups"), "filters are not implemented for security groups and will be removed in a future release"))
```

`should not` isn't a strong enough wording.

Reply:
Changes done.